builtin/providers/openstack/resource_openstack_compute_instance_v2.go

```diff
@@ -998,7 +998,7 @@ func getInstanceAccessAddresses(d *schema.ResourceData, networks []map[string]in
 			hostv6 = n["fixed_ip_v6"].(string)
 		}
 
-		if n["access_network"].(bool) {
+		if ok, an := n["access_network"].(bool); ok && an {
```

Comment:
Yes indeed. I should really just step away from any keyboard for the rest of the day.